wgpu-hal/src/gles/adapter.rs

```diff
@@ -909,7 +1061,11 @@ impl crate::Adapter<super::Api> for super::Adapter {
 
             Some(crate::SurfaceCapabilities {
                 formats,
-                present_modes: vec![wgt::PresentMode::Fifo], //TODO
+                present_modes: if cfg!(windows) {
+                    vec![wgt::PresentMode::Fifo, wgt::PresentMode::Mailbox]
```

Comment:
If you turn off vsync, do you get tearing? If so, this should be Fifo + Immediate; not mailbox.

```suggestion
                    vec![wgt::PresentMode::Fifo, wgt::PresentMode::Mailbox]
```

Reply:
If you turn off the compositor on older version of Windows, you might manage some tearing.

Reply:
Ah, then lets leave it as mailbox.